recipes/deepac/meta.yaml

```diff
@@ -0,0 +1,58 @@
+{% set name = "deepac" %}
+{% set version = "0.9.1" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: dc6976b26bb46d8b1f725ab1202e3862ea405d3c664df4cc3a487d4f66d6cdd8
+
+build:
+  noarch: python
+  number: 0
+  entry_points:
+    - deepac=deepac.command_line:main
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vvv "
+
+requirements:
+  host:
+    - biopython
+    - h5py
+    - keras >=2.2.4
+    - matplotlib
+    - numpy >=1.15.4
+    - pip
+    - python >=3, <3.7
+    - scikit-learn
+    - tensorflow >=1.12
+  run:
+    - biopython
+    - h5py
+    - keras >=2.2.4
+    - matplotlib
+    - numpy >=1.15
+    - python >=3, <3.7
+    - scikit-learn
+    - tensorflow >=1.12
+
+test:
+  imports:
+    - deepac
+    - deepac.eval
+  commands:
+    - deepac --help
+
+about:
+  home: https://gitlab.com/rki_bioinformatics/DeePaC
+  license: MIT
+  license_family: MIT
+  license_file: 
+  summary: Predicting pathogenic potentials of novel DNA with reverse-complement neural networks.
+  doc_url: 
```

Comment:
please fill these out as well or remove, the dev_url

https://gitlab.com/rki_bioinformatics/DeePaC